### PR TITLE
javascript-and-json exercise.-Dylan De Vei

### PR DESCRIPTION
**Pull request by dylandevei**
Screen Shot 2022-04-01 at 10 39 18 AM

### CODE REVIEW

File: javascript-and-json/main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+];
+
+console.log('books:', books);
+console.log('type of books:', typeof (books));
+
+var jsonStringify = JSON.stringify(books);
+console.log('JSON.Stringify:', jsonStringify);
+console.log('type of JSON.stringify:', typeof (jsonStringify));
+
+var student = '[{"numberID": 123452}]';
```
Comment (codingcodymiller) [inline javascript-and-json/main.js]: The name of the property should just be `id`, not `numberID`. Also, you are missing the `name` property

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by dylandevei** [comment]
Screen Shot 2022-04-01 at 10 47 40 AM
here are those changes, thank you!

File: javascript-and-json/main.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+];
+
+console.log('books:', books);
+console.log('type of books:', typeof (books));
+
+var jsonStringify = JSON.stringify(books);
+console.log('JSON.Stringify:', jsonStringify);
+console.log('type of JSON.stringify:', typeof (jsonStringify));
+
+var student = '[{ "id": 123452}, {"name": "Dylan De Vei"} ]';
```
Comment (codingcodymiller) [inline javascript-and-json/main.js]: This is an array with two objects, this should be one object with two properties

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-and-json/main.js (hunk 2 above)
Comment (dylandevei) [reply javascript-and-json/main.js]: Whoops, my mistake, will change now

**Comment by dylandevei** [comment]
Screen Shot 2022-04-01 at 10 53 08 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**